tests/components/matter/test_sensor.py

```diff
@@ -64,6 +64,16 @@ async def temperature_sensor_node_fixture(
     )
 
 
+@pytest.fixture(name="eve_contact_sensor_node")
```

Comment:
This fixture is already available to the whole test package. We don't need to define it again. Just request it in the test.